setup.py

```diff
@@ -24,7 +24,7 @@
 
 INSTALL_REQUIRES = [
     'metomi-rose>=2.0a1',
-    'cylc-flow>=8.0a3',
+    'cylc-flow>=8.0a*',
```

Comment:
Eh?

```suggestion
    'cylc-flow>=8.0a3',
```

Was this because the `.dev` was causing install issues?